azerothcore/azerothcore-wotlk
# Fix(Core/Script): Molten Core: Gehennas

Co-Author: @sanctum32 (Code)
Co-Author: @T1ti (Research)
Co-Author: @Hacki95 (Coordination)



## Changes Proposed:
- Adjust Gehennas Timers
- Adjust the timers for the spells from the adds
- Split Shadow Bolt into its two spells (One of the main aggro target and one for a random target)

## Issues Addressed:

- Closes #5458

## SOURCE:


## Tests Performed:

- Applied Patch
- Fought Gehennas


## How to Test the Changes:


1. Apply Patch
2. .tele moltencore
3. Fight Gehennas
4. Check the timers
5. To check Shadow Bolt you need to test it with several players.

## Known Issues and TODO List:


-/


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
